meson.build

```diff
@@ -155,9 +155,14 @@ if is_parallel_build
 
   # find mpi
   if is_mpich
-    mpi = dependency('mpich', required : true)
-    mpifort = dependency(mpifort_name, required : mpi.found())
-    dependencies += [ mpi, mpifort ]
+    if is_cray
+      mpi = dependency('mpich', required : true)
+      mpifort = dependency(mpifort_name, required : mpi.found())
+      dependencies += [ mpi, mpifort ]
```

Comment:
Hi @jdhughes-usgs, I was just wondering if and why that mpi dependency on Cray is actually needed